Source/FortranInterface/WarpX_picsar.F90

```diff
@@ -1,20 +1,16 @@
 #if (AMREX_SPACEDIM == 3)
 
-#define WRPX_PXR_GETEB_ENERGY_CONSERVING  geteb3d_energy_conserving_generic
 #define WRPX_PXR_CURRENT_DEPOSITION       depose_jxjyjz_generic
 
 #elif (AMREX_SPACEDIM == 2)
 
-#ifdef WARPX_RZ
+#ifdef WARPX_DIM_RZ
 
-#define WRPX_PXR_GETEB_ENERGY_CONSERVING  geteb2drz_energy_conserving_generic
 #define WRPX_PXR_CURRENT_DEPOSITION       depose_jrjtjz_generic_rz
 #define WRPX_PXR_RZ_VOLUME_SCALING_RHO    apply_rz_volume_scaling_rho
```

Comment:
Could you also implement `apply_rz_volume_scaling_rho` in C++, as part of this PR?
It seems that this would be similar to the C++ version of `ApplyInverseVolumeScalingToCurrentDensity`.

Reply:
I can do that. I was going to wait until the charge deposition was converted since they are related. Is anyone working on that?

Reply:
Ah, ok, makes sense. Please ignore my comment, then.

Reply:
@dpgrote I do not know of anyone working on that. I added it in the fortran-to-c++ issue.

Reply:
I'll do the conversion and put it in a separate PR